pootle/apps/pootle_misc/checks.py

```diff
@@ -266,56 +267,82 @@ class SkipCheck(Exception):
     pass
 
 
-class ENChecker(checks.TranslationChecker):
+class ENChecker(checks.UnitChecker):
+
+    def __init__(self, checkerconfig=None, excludefilters=None,
+                 limitfilters=None, errorhandler=None):
+        super(ENChecker, self).__init__(checkerconfig, excludefilters,
+                                        limitfilters, errorhandler)
+
+    def run_test(self, test, unit):
+        """Runs the given test on the given unit.
+
+        Overrides `TranslationChecker`'s method to avoid extra plural handling
```

Comment:
`TranslationChecker` is no longer used, so it is not being overridden. Please adjust this docstring.